src/index.ts

```diff
@@ -248,14 +353,14 @@ export function platformApi<
     Platform extends SupportedPlatform,
     RunTarget extends SupportedRunTarget<Platform>,
     Capabilities extends SupportedCapability<Platform>[]
->(options: PlatformApiOptions<Platform, RunTarget, Capabilities>): PlatformApi<Platform, RunTarget> {
+>(options: PlatformApiOptions<Platform, RunTarget, Capabilities>): PlatformApi<Platform, RunTarget, Capabilities> {
     switch (options.platform) {
         case 'android':
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            return androidApi(options as any) as PlatformApi<Platform, RunTarget>;
+            return androidApi(options as any) as any;
```

Comment:
Why? Does this not work anymore or do you just cast as `any` because it doesn’t matter anyways?

Reply:
Both. Adding the additional `Capabilities` generic broke this again, and rather than spend hours trying to find a cast that does work, I opted for this because it really, _really_ doesn't matter in the slightest.